### DISCUSSION TITLE
External subtitles file .srt resulting fragment subtitles does not fit the cut

### DISCUSSION THREAD

**Discussion by slrslr** [mifi/lossless-cut#1200] [Q&A] [answered]
Hello,
i have added .mp4 file (that contains no subtitles) to LC 3.45.0 (AppImage), then drag & drop .srt file that fits the video file, made a cut/fragment file and when playing this fragment/cut file, i see it shows subtitles from the begining of the .srt file, not from the part that would fit my cut timeframe.

What i am doing wrong please and how to proceed? #907 @mifi 

Some [detail about the resulting file provided by ffplay](https://paste.nolog.cz/?f285214def0922a6#5Gfk5dv4uPrDpNZP3vAVmCjubWpu5VvtGorwwfzDLC52).

**Comment by mifi** [comment, marked as answer]
Hi. Are you doing the two operations of adding srt and cutting in the same operation? If so, that's not currently supported and you need to do it in separate operations. See #896

**Reply by slrslr** [reply]
Thanks for the help. Possibly that is the reason. Maybe in future, it will be possible for LC to first insert subtitles and in second steps do the cutting.

For those with same issue, here are some ideas about **workaround**:

Note that it may not work for some video files

**A) Using LC:**
Drag&drop video file to LC
drag&drop subtitle file to LC
Export (mp4 in my case)
(seeing 3 tracks are there now)
drag&drop file that has the subtitle track inside it now
cut
export
subtitles fits the video now

**B) using ffmpeg:**
`ffmpeg -i "InputVideo" -i "InputSubs.srt" -c:v copy -c:a copy -c:s mov_text "OutputVideoWithSubs"`
drag&drop OutputVideoWithSubs to LC
cut
export
subtitles fits the video now

**C) using mkvtoolnix GUI app**
drag&drop video to MKVT
drag&drop subtitle to MKVT
start muxing
drag&drop OutputVideoWithSubs to LC
cut
export
subtitles fits the video now

ISSUE - subtitles are wrong in merged file:
After merging the cuts/fragments, resulting file may play subtitles from beginning not when person start speaking. Fix? Extract subtitles from resulting merged file using command "ffmpeg -i "pathtofile" subtitles.srt" then open these in Aegisub or other subtitle editor, click first subtitle, click menu "Timing"->Shift times-> shift by time, forward, selection onward, start and end times. The time you will set here will be the number of seconds from begining to when person first start talking. Save and rename subtitle file to match video or add subtitles to a file using: ffmpeg -i input.mp4 -vf "subtitles=subtitle.srt" output.mp4
